pkg/extensions/search/userprefs_test.go

```diff
@@ -818,17 +817,17 @@ func TestGlobalSearchWithUserPrefFiltering(t *testing.T) {
 func TestExpandedRepoInfoWithUserPrefs(t *testing.T) {
 	Convey("ExpandedRepoInfo with User Prefs", t, func() {
 		dir := t.TempDir()
-		port := GetFreePort()
-		baseURL := GetBaseURL(port)
+		port := test.GetFreePort()
+		baseURL := test.GetBaseURL(port)
 		conf := config.New()
 		conf.HTTP.Port = port
 		conf.Storage.RootDirectory = dir
 
 		simpleUser := "simpleUser"
 		simpleUserPassword := "simpleUserPass"
-		credTests := fmt.Sprintf("%s\n\n", getCredString(simpleUser, simpleUserPassword))
+		credTests := fmt.Sprintf("%s\n\n", test.GetCredString(simpleUser, simpleUserPassword))
```

Comment:
fmt.Sprintf() may no longer be necessary, if test.GetCredString() already adds `\n`